### PR TITLE
Add general page

### PR DESCRIPTION
**Pull request by kalachou**

### CODE REVIEW

File: src-be/app/service/radars.service.ts [hunk 1]
```diff
@@ -31,14 +31,23 @@ export class RadarsService {
...
+	public async getAllLatestRadars(): Promise<RadarEntity[]> {
+		const latestRadars: RadarEntity[] = [];
+
+		for (const radar of radars.values()) {
+			latestRadars.push(radar[radar.length - 1]);
+		}
+		return Promise.resolve(latestRadars);
+	}
+
 	public async getRadarsById(radarId: string): Promise<RadarEntity[]> {
```
Comment (vladimir-sviridenko) [inline src-be/app/service/radars.service.ts:43]: name it like getRadarById

File: src-fe/src/app/libs/radars-general-view/radars-general-view.component.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
+@import 'variables';
+
+:host {
```
Comment (vladimir-sviridenko) [inline src-fe/src/app/libs/radars-general-view/radars-general-view.component.scss:3]: apply css comb

File: src-be/app/service/radars.service.ts (hunk 1 above)
Comment (kalachou) [reply src-be/app/service/radars.service.ts:43]: isn't related to this PR=)

**Review by vladimir-sviridenko — APPROVED** [review]

File: src-fe/src/app/libs/radars-general-view/radars-general-view.component.scss (hunk 2 above)
Comment (kalachou) [reply src-fe/src/app/libs/radars-general-view/radars-general-view.component.scss:3]: nice tool, but currently it conflicts with stylelint and therefore husky pre-commit. cannot be applied directly

**Comment by kalachou** [comment]
The commits of the PR are merged in [](https://github.com/radars-app/radars-app/pull/57)

**Pull request closed without merging**